remotebackup.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+
+#This script connects  to a remote server via ssh, creates a .tar.gz backup, and uses scp to secure copy it to the host computer. 
```

Comment:
Look at the suggested format I gave in the other comment and follow that - more standard and more legible